ppgan/datasets/transforms/transforms.py

```diff
@@ -72,3 +74,61 @@ def _apply_image(self, image):
         if self.params['flip']:
             return F.hflip(image)
         return image
+
+
+@TRANSFORMS.register()
+class Add(T.BaseTransform):
+    def __init__(self, value, keys=None):
```

Comment:
Better to add commens. Although, there is no comments in the existing code. We will add in future